Manatee.Trello/Rest/WebApiClient.cs

```diff
@@ -176,11 +177,23 @@ private static HttpContent GetContent(WebApiRestRequest request)
 					formData.Add(content, $"\"{parameter.Key}\"");
 				}
 
-				var byteContent = new ByteArrayContent(request.File);
-				formData.Add(byteContent, "\"file\"", $"\"{request.FileName}\"");
-				TrelloConfiguration.Log.Debug($"\tContent: {formData}");
-
-				return formData;
+                if (File.Exists(request.FilePath))
```

Comment:
Above, it's checking `request.File != null`, but here your checking the file path.  Aren't these going to be mutually exclusive?

Reply:
Yeah, actually you are right it is mutually exclusive